app/javascript/stylesheets/home.scss

```diff
@@ -0,0 +1,83 @@
+.hero-img {
+  background-image: linear-gradient(
+      rgba(37, 38, 54, 0.2),
+      rgba(37, 38, 54, 0.2)
+    ),
+    url("./../images/banner.svg");
+  height: 720px;
+  background-position: left;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.home-title {
+  color: #252636;
+  font-weight: 600;
+  font-size: 32px;
+}
+
+.new-classes {
+  margin-left: 9em;
+  padding-top: 7em;
+
+  .filter-wrapper {
+    width: 100%;
+    display: flex;
+    overflow-x: auto;
+    margin: 0.75rem 0;
+  }
+
+  .new-classes-wrapper {
+    width: 100%;
+    display: flex;
+    overflow-x: auto;
+
+    &::-webkit-scrollbar {
+      width: 0;
+    }
+  }
+}
+
+.categories {
+  margin-left: 9em;
+  padding-top: 7em;
+
+  .categories-wrapper {
+    display: grid;
+    grid-gap: 1rem;
+  }
+}
+
+.popular {
+  margin-left: 9em;
+  padding-top: 7em;
+}
+
+@media screen and (max-width: 960px) {
+  .home-title {
+    font-size: 21px;
+  }
+  .hero-img {
+    height: 550px;
+  }
+
+  .new-classes {
+    margin-left: 1em;
+    padding-top: 5em;
+  }
+
+  .categories {
+    margin-left: 1em;
+    padding-top: 5em;
+
+    .categories-wrapper {
+      grid-template-columns: repeat(2, 10.25rem);
```

Comment:
**praise**: nice! grid is amazing isn't it 😍

Reply:
yes it is 😄